python/llm/src/bigdl/llm/vllm/core/scheduler.py

```diff
@@ -101,6 +117,8 @@ def __init__(
         self.cleaned: List[int] = []
         self.kv_cache = kv_cache
         # Co(gc): We no longer have the swapped space as we are not deciding which to swap
```

Comment:
We may want to delete this comment since we now having a swap area

Reply:
Currently maybe not. We have only implement recompute right now and swap is still unavailable.